v2/guides/blocks/recipe-social-login-info.mdx

```diff
@@ -0,0 +1,5 @@
+
+The example app includes all the supported third party providers in the config files.
+Just remove the ones that you do not want to use.
+
+You will also have to update the credentials for each provider. For more details on how to do this check our [extensive instructions](/docs/thirdparty/pre-built-ui/setup/backend#3-initialise-social-login-providers).
```

Comment:
this text shows up even for multi tenancy selection. In case of mta, the thirdparty.init is empty, and instead of saying this, we need to say that the social / enterprise / SAML login providers are configured during tenant creation: https://test.supertokens.com/docs/thirdpartyemailpassword/common-customizations/multi-tenancy/new-tenant-config